packages/decap-cms-core/src/components/Editor/EditorPreviewPane/EditorPreviewPane.js

```diff
@@ -6,8 +6,8 @@ import ImmutablePropTypes from 'react-immutable-proptypes';
 import Frame, { FrameContextConsumer } from 'react-frame-component';
 import { lengths } from 'decap-cms-ui-default';
 import { connect } from 'react-redux';
-import { encodeEntry } from 'decap-cms-lib-util/src/stega';
 
+import { encodeEntry } from '../../../lib/stega';
```

Comment:
Thanks, this will fix #7401!

One thing might be worth doing is to add an alias to each `tsconfig.json`, e.g.
```json
"compilerOptions": {
  "paths": {
    "@/*": ["./src/*"]
  }
```

then here you can use the alias path...
```js
import { encodeEntry } from '@/lib/stega';
```

Personally, I find this easier to rationalize the file structure, when importing higher up the tree. This is definitely not required, but something to consider.

Thanks again for your awesome contributions!

Reply:
Thanks @eoneill, I was not aware that this was causing runtime errors! I created a separate PR (#7405) to address this in isolation.